packages/react-native/src/primitives/Checkbox/__tests__/Checkbox.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+
+import Checkbox from '../Checkbox';
+
+const onChange = jest.fn();
+
+describe('Checkbox', () => {
+  [true, false].forEach((value) => {
+    it(`renders as expected when selected is ${value}`, () => {
+      const checkbox = create(
+        <Checkbox selected={value} value={value} onChange={onChange} />
+      );
+      expect(checkbox.toJSON()).toMatchSnapshot();
+    });
+  });
+
+  it('renders as expected when disabled', () => {
```

Comment:
Can we also check that the `selected` state cannot be toggled when the `Checkbox` is disabled?